From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] Build fixes for armel.

Fix build failures on armel. Thanks to Michael Casadevall for the patch.

Signed-off-by: Modestas Vainius <modestas@vainius.eu>

---
 src/EngineController.cpp        |    4 ++--
 src/widgets/TokenWithLayout.cpp |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/EngineController.cpp b/src/EngineController.cpp
index b402e1f..3457140 100644
--- a/src/EngineController.cpp
+++ b/src/EngineController.cpp
@@ -460,12 +460,12 @@ EngineController::decreaseVolume( int ticks ) //SLOT
 int
 EngineController::setVolume( int percent ) //SLOT
 {
-    percent = qBound( 0, percent, 100 );
+    percent = qBound<qreal>( 0, percent, 100 );
     m_volume = percent; 
     
     // Phonon stays completely mute if the volume is lower than 0.05, so we shift and limit the range 
     qreal newVolume =  ( percent + 4 ) / 100.0;
-    newVolume = qBound( 0.04, newVolume, 1.0 );
+    newVolume = qBound<qreal>( 0.04, newVolume, 1.0 );
     m_audio->setVolume( newVolume );
     
     AmarokConfig::setMasterVolume( percent );
diff --git a/src/widgets/TokenWithLayout.cpp b/src/widgets/TokenWithLayout.cpp
index 0b84698..eb33f94 100644
--- a/src/widgets/TokenWithLayout.cpp
+++ b/src/widgets/TokenWithLayout.cpp
@@ -136,11 +136,11 @@ void TokenWithLayout::fillMenu( QMenu * menu )
                 }
             }
 
-            int max = qMax( spareWidth, 0.0 );
+            int max = qMax<qreal>( spareWidth, 0.0 );
             debug() << "slider max value: " << max;
 
             if ( max >= m_width * 100.0 )
-                slider->setMaximum( qMax( spareWidth, 0.0 ) );
+                slider->setMaximum( qMax<qreal>( spareWidth, 0.0 ) );
             else
                 slider->setMaximum( m_width * 100.0 );
         }
-- 
tg: (d5becae..) general/armel_ftbfs_fix (depends on: upstream)
